lib/resty/healthcheck.lua

```diff
@@ -1275,7 +1287,9 @@ local defaults = {
       type = "http",
       timeout = 1,
       concurrency = 10,
+      http_method = "GET",
```

Comment:
should use `request_method`

same with nginx name style

<img width="499" alt="image" src="https://github.com/user-attachments/assets/86e767f4-fe63-4ee4-9445-95e508990b08" />